### ISSUE TITLE
Writing a Lucee config file removes the predefined Lucee mappings

### ISSUE THREAD

**Issue by jcberquist** [bdw429s/cfconfig#1]
I noticed that when reading in mappings from a Lucee config XML file, cfconfig purposely ignores the mappings that Lucee predefines for you. (e.g. `/lucee-server/`). But when the config is written back to the file, it does not currently have a check to keep those predefined mappings in place, and so they get removed from the XML file.

**Comment by bdw429s** [comment]
Ahh yes, thank you for noticing this.  Originally when I started, I didn't clear out the existing mappings so it wouldn't remove those inbuilt ones.  Later on, I realized that I could delete a mapping since it only ever added new mappings so I changed it to wipe out existing mappings and then write them back in.  I'll have to re-think that part to make sure we can remove user mappings without losing the inbuilt ones.

**Comment by bdw429s** [comment]
OK, I think I have this fixed for Lucee.  Please check and confirm.  If it works correctly, I'll fix the Adobe providers as well since they have their own built in mappings.

**Comment by bdw429s** [comment]
Fixed for Adobe servers as well.

**Comment by jcberquist** [comment]
It works perfectly, thanks! This is a great project. Now I would love to see the log file config included ;)

**Comment by bdw429s** [comment]
@jcberquist I will gladly merge your pull requests!  Just keep in mind that the log functionality for both Lucee and Adobe needs to be inspected so we can find a generic storage solution that applies to both of them.

**Comment by jcberquist** [comment]
Hi @bdw429s, sorry to resurrect this old issue - I have finally circled back to looking at logging again. I added support for Lucee logger configuration here: https://github.com/jcberquist/cfconfig/commit/916ba194d0a17f9a72be759cdba9c45ee280b9f7 

I did not add anything for Adobe CF since the approach CF takes to logger settings seems different to me. Looking at the `neo-logging.xml` files for CF it looks like log settings are specified globally for all log files and not for one log file at a time. So while I get that a generic solution would be preferable, I am not sure how to do it here. 

I do want to see support for the logging settings added, so I would welcome your feedback and guidance to get this done. Thanks!
